qiskit/transpiler/target.py

```diff
@@ -773,6 +807,55 @@ def check_obj_params(parameters, obj):
                     )
         return False
 
+    def calibration_supported(
```

Comment:
I prefer calling this `has_calibration` is that ok for you?

Reply:
Sure. I changed the name in [bdb790a](https://github.com/Qiskit/qiskit-terra/pull/9587/commits/bdb790af888455c2f31d5fa7d111a9d697798796)